scene/realistic-light-and-shadows/src/main/java/com/esri/samples/realistic_lighting_and_shadows/RealisticLightingAndShadowsController.java

```diff
@@ -0,0 +1,203 @@
+/*
+ * Copyright 2020 Esri.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.esri.samples.realistic_lighting_and_shadows;
+
+import com.esri.arcgisruntime.mapping.view.LightingMode;
+import javafx.fxml.FXML;
+import com.esri.arcgisruntime.layers.ArcGISSceneLayer;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.control.Slider;
+import javafx.util.StringConverter;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.GregorianCalendar;
+import java.util.TimeZone;
+
+import com.esri.arcgisruntime.mapping.ArcGISScene;
+import com.esri.arcgisruntime.mapping.ArcGISTiledElevationSource;
+import com.esri.arcgisruntime.mapping.Basemap;
+import com.esri.arcgisruntime.mapping.Surface;
+import com.esri.arcgisruntime.mapping.view.AtmosphereEffect;
+import com.esri.arcgisruntime.mapping.view.SceneView;
+import com.esri.arcgisruntime.mapping.view.Camera;
+
+public class RealisticLightingAndShadowsController {
+
+  @FXML
+  private SceneView sceneView;
+  @FXML
+  private Label time;
+  @FXML
+  private Slider timeSlider;
+  @FXML
+  private Button noSunButton;
+  @FXML
+  private Button sunOnlyButton;
+  @FXML
+  private Button sunAndShadowsButton;
+  private Surface surface;
+  private Calendar calendar;
+
+  public void initialize() {
+    try {
+
+      // create a scene and add a basemap to it
+      ArcGISScene scene = new ArcGISScene();
+      scene.setBasemap(Basemap.createTopographic());
+
+      // add the SceneView to the stack pane
```

Comment:
What's actually happening in the line below is that we're adding the ArcGISScene to our scene view, not the scene view to the stack pane, so let's update this comment:
```suggestion
      // add the scene to the scene view
```